README.md

```diff
@@ -60,5 +60,70 @@ To register a device, run as root from LmP device:
 (i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+In scenarios where you'd like to test the API call without involving a physical device or emulation:
+
+This project includes a simple [fake-lmp-device-register](./fake-lmp-device-register) tool. 
+This tool mimics the behavior of `lmp-device-register`, enabling you to quickly test this project's API 
+without a real device.
+
+## Running the fake-lmp-device-register
+
+### Dependencies:
+- Python 3
+- `requests` library
+- `openssl` command-line tool
+- `curl`
+
+### Usage:
+
+1. Install the required Python library using pip:
+
+```shell
+pip3 install requests
+```
```

Comment:
If we are in the business to give detailed install instructions, we'd need to start with installing Python and OpenSSL.
Then we'd need to decide which platforms we support, and give instructions for every platform.
But just IMHO our users are smart enough to know how to install things; we've given a full list of dependencies already.

All that long text is just to reason that it's better to remove this paragraph.